lib/locales/pl.ts

```diff
@@ -359,13 +359,13 @@ export const locale: Locale = {
     margaret_of_scotland: 'Św. Małgorzaty Szkockiej',
     maria_goretti_virgin: 'Św. Marii Goretti, dziewicy i męczennicy',
     marie_eugenie_of_jesus_milleret_de_brou_virgin:
-      'Św. Marii Eugenii od Jezusa Milleret de Brou, dziewicy i założycielky',
+      'Św. Marii Eugenii od Jezusa Milleret de Brou, dziewicy i założycielki',
     mark_evangelist: 'Św. Marka, Ewangelisty',
-    martha_of_bethany_mary_of_bethany_and_lazarus_of_bethany: 'Św. Marty, Marii i Łazarza',
+    martha_of_bethany_mary_of_bethany_and_lazarus_of_bethany: 'Świętych Marty, Marii i Łazarza',
```

Comment:
All celebrations have the canonisation title shortened; why this one not? :thinking:

Reply:
`Św.` means `Święty` (singular). When there are more than one saint `Świętych` (plural) is used. The convention is already followed in the file - check out entry for `Basil the Great and Gregory Nazianzen` or `Joachim and Anne`.

Reply:
> `Św.` means `Świętego` (singular).

I understand this. :wink:

> When there are more than one saint `Świętych` (plural) is used. The convention is already followed in the file - check out entry for `Basil the Great and Gregory Nazianzen` or `Joachim and Anne`.

For some zome reason, I failed to notice this. I need to note this in the Polish l10n rules (not yes published).

> The proper way to abbreviate plural `Świętych` in polish language would be `Św.Św. Marty, Marii i Łazarza` which looks weird and nobody is using that (maybe except for linguists).

What?! 👀 That indeed looks weird. I have never ever seen a language where they would double the shortening (in Slovak, we use `sv.`/`bl.` for both singular and plural; in English, an `s` is appended; in Latin, the first letter is doubled, i.e. `S.` → `Ss.`, `B.` → `Bb.`).